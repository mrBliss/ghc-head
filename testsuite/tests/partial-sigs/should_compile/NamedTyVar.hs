{-# LANGUAGE PartialTypeSignatures #-}
module NamedTyVar where

foo :: (_a, b) -> (a, _b)
foo (x, y) = (x, y)
