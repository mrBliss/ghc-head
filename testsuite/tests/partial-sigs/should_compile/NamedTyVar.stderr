TYPE SIGNATURES
  foo :: forall b a. (a, b) -> (a, b)
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
